lib/galaxy/managers/secured.py

```diff
@@ -143,3 +143,25 @@ def filter_owned(self, user, **kwargs):
         """
         # just alias to list_owned
         return self.list_owned(user, **kwargs)
+
+    def get_mutable(self, id: int, user: Optional[model.User], **kwargs: Any) -> Any:
+        """
+        Return the item with the given id if the user can mutate it,
+        otherwise raise an error. The user must be the owner of the item.
+
+        :raises exceptions.ItemOwnershipException:
+        """
+        item = self.get_owned(id, user, **kwargs)
+        return self.error_unless_mutable(item)
+
+    def error_unless_mutable(self, item):
+        """
+        Raise an error if the item is NOT mutable, otherwise return the item.
```

Comment:
Do we need to return an item here? The only place where the return value is used is the `get_mutable` method, and it doesn't have to. The rest of the calling code uses it as a check and doesn't care for a return value.

Reply:
No, is not really necessary, I was just following the pattern of `error_unless_owner`. I can turn this into a check method only.

Reply:
Yeah, I see. Both are fine of course, but my vote would be for breaking the pattern (i.e., simplifying galaxy's code base one tiny step at a time :) Besides, there are 2 more similar methods (in `rbac_secured.py` and `datasets.py`) which do not return a value - so it's already not consistent.